mypy/nodes.py

```diff
@@ -465,18 +473,17 @@ def accept(self, visitor: StatementVisitor[T]) -> T:
 class ImportAll(ImportBase):
     """from m import *"""
 
-    __slots__ = ("id", "relative", "imported_names")
+    __slots__ = ("id", "relative")
+
+    __match_args__ = ("id", "relative")
 
     id: str
     relative: int
-    # NOTE: Only filled and used by old semantic analyzer.
```

Comment:
I think that removing old parts can go into a separate PR. It is not related to `__match_args__`.